if (true)
	try {
		throw new Exception();
	} catch (Exception e) {
	}